src/Microsoft.AspNetCore.Razor.Language/CodeGeneration/DefaultDocumentWriter.cs

```diff
@@ -70,11 +70,16 @@ public void RenderChildren(RazorIRNode node)
             public override void VisitDocument(DocumentIRNode node)
             {
                 RenderChildren(node);
+
+                Context.Writer.WriteLine("#pragma warning restore 1591");
             }
 
             public override void VisitChecksum(ChecksumIRNode node)
             {
                 Context.BasicWriter.WriteChecksum(Context, node);
+
+                Context.Writer.WriteLine("// <auto-generated/>");
```

Comment:
Actually I am going to revert this because we only need this for design time in this branch.